### ISSUE TITLE
onPress handler does not receive event

### ISSUE THREAD

**Issue by mtford90** [ide/react-native-button#5]
Hi there,

My onPress handlers are not receiving events - I'm using the buttons within a ListView:

```
var SettingsTab = React.createClass({
  getInitialState: function () {
    var ds = new ListView.DataSource({rowHasChanged: (r1, r2) => r1 !== r2});
    return {
      menuItems: ds.cloneWithRows(['Exercises']),
      menuRoutes: {
        Exercises: {
          title: ''
        }
      }
    }
  },
  _handleButtonPress: function (e) {
    console.log('e', e);
  },
  render: function () {
    return (
      <View style={{
        backgroundColor: '#414A8C',
        flex: 1,
        alignItems: 'center',
        paddingTop: 64
        }}>
        <ListView
          dataSource={this.state.menuItems}
          automaticallyAdjustContentInsets={false}
          renderRow={(rowData) => <Button onPress={this._handleButtonPress}>{rowData}</Button>}
          />
      </View>
    );
  }
});
```

In the above, e is `undefined`.

Any ideas?

Cheers!

**Comment by ryanmcdermott** [comment]
I'm having the same issue.

P.S. Very cool component plugin! @ide

**Comment by ide** [comment]
This is happening because TouchableOpacity doesn't pass up the event so there's no way for Button to pass it along. Can you file a bug with React Native about this?

**Comment by ide** [comment]
This will be fixed in React Native 0.11-rc.
